.github/workflows/build.yml

```diff
@@ -27,8 +27,11 @@ jobs:
           PATH: src
 
       - name: Build and test
-        env:
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         run: |
           tox
         shell: bash
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        with:
+          fail_ci_if_error: true
```

Comment:
i think these should actually be left as the default of false - if my understanding is correct then this means it will fail if the upload to codecov fails, which we don't really care about and may cause problems (e.g if it causes a release to fail even though coverage levels were OK and build and everything succeded). can you remove fail_ci_if_error from everywhere?